docs/source/downstream_tasks.rst

```diff
@@ -0,0 +1,292 @@
+Downstream tasks
+================
+
+PBG is intended to be a tool that takes as input a graph and produces as output embeddings for it.
```

Comment:
Lines 1-16 read as a philosophical statement ("it *should* be", "does not intend to"), which don't belong in the docs.

I think this can be made into a short descriptive paragraph:

PBG is a tool for producing graph embeddings, that is it takes a graph (i.e. an edgelist) as input and produces embeddings for each node [entity] in the graph. These embeddings can be used in a variety of ways to solve downstream tasks. Below, we provide example code for how to perform several common downstream tasks with PBG embeddings. In some cases, other open source tools can be used to perform these downstream tasks (e.g. nearest neighbor search) with PBG embeddings, which we discuss below.